docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/src/uploadtoftp/Module.md

```diff
@@ -1 +1,92 @@
-# Convert JSON to XML and upload to FTP
+Template for JSON to XML and upload to FTP
+
+# REST to SOAP 
+
+This is a template for the [JSON to XML and upload to FTP tutorial](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario. 
+
+
+## Using the Template
+
+Run the following command to pull the `uploadtoftp` template from Ballerina Central.
+
+```
+$ ballerina pull wso2/uploadtoftp
+```
+
+Create a new project.
+
+```bash
+$ ballerina new converting-json-to-xml-and-upload-to-ftp
+```
+
+Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
+
+```bash
+$ ballerina add -t wso2/uploadtoftp uploadtoftp
+```
+
+This automatically creates restful_service for you inside the `src` directory of your project.  
```

Comment:
Should be something like converting_json_to_xml_and_upload_to_ftp.

Reply:
Changed to upload_to_ftp